src/styles/main.scss

```diff
@@ -1,3 +1,74 @@
-body {
+$day-cell-size: 100px;
+$day-margin: 0.5px;
+$day-bckgrd-color: #eee;
+$day-border-width: 1px;
+$days-of-week: mon 1, tue 2, wed 3, thu 4, fri 5, sat 6, sun 7;
+$calendar-columns: 7;
+$calendar-gap: 1px;
+$calendar-padding: 10px;
+$calendar-width: ($day-cell-size * $calendar-columns)
+  + ($calendar-gap * $calendar-columns)
+  + ($calendar-padding * 2);
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
```

Comment:
it's not a good practice to use styling with * selector. It's allowed to use only box-sizing: border-box at this selector